index.js

```diff
@@ -12,16 +12,26 @@ module.exports = function(content) {
 	this.cacheable && this.cacheable();
 
 	var query = loaderUtils.getOptions(this);
+	var configKey = query.config || "svgAsSymbolLoader";
```

Comment:
I removed this logic, I think `loaderUtils#getOptions` handles everything needed.